pages/portfolios/_id.vue

```diff
@@ -0,0 +1,15 @@
+// simple page, shows the url slug
```

Comment:
The comment can be removed I suppose. People working with nuxt know what the file does.

Reply:
Comment now removed.